scripts/bootloader/generate_zip.py

```diff
@@ -21,6 +21,8 @@ def parse_args():
     parser.add_argument('--output', required=True, type=argparse.FileType(mode='w'), help='Output zip path')
     parser.add_argument('--bin-files', required=True, type=argparse.FileType(mode='r'), nargs='+',
                         help='Bin files to be stored in zip')
+    parser.add_argument('--fwversion', required=True)
```

Comment:
Could you add some help text to these? Especially the logic regarding `name` would be helpful.